content/docs/controlled-input-null-value.md

```diff
@@ -0,0 +1,19 @@
+---
+id: controlled-input-null-value
+title: Controlled input null value
+permalink: docs/controlled-input-null-value.html
+---
+
+Specifying the value prop on a [controlled component](/docs/forms.html) prevents the user from changing the input unless you desire so.
+
+You might have run into a problem where value is specified, but the input can still be changed without consent. In this case, you might have accidentally set value to undefined or null.
+
+For example, this code shows this phenomenon; after a second, the text becomes editable.
```

Comment:
Grammatical nit: There's a lot of unnecessary commas in this content. Maybe we could change it something like?

> You might have encountered a situation where `value` is specified but an input is still editable. In this case you might have accidentally set `value` to `undefined` or `null`.

> The following code demonstrates this. (After a short delay, the input becomes editable.)